lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -160,8 +160,9 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const thumbnailResult = thumbnailAudit && thumbnailAudit.result;
     if (thumbnailResult && thumbnailResult.details) {
       timelineEl.id = thumbnailResult.id;
+      // @ts-ignore TODO(bckenny): fix detailsRenderer.render input type
```

Comment:
input type? output type looks busted too based on the next line change 😆

Reply:
> input type? output type looks busted too based on the next line change

haha, maybe both then